dataset/companies/FNB.yml

```
@@ -0,0 +1,8 @@
+ - Name: "McDonald"
```

```suggestion
 - Name: "McDonalds"
```
They use the plural.